src/www/js/app.js

```diff
@@ -23,14 +23,33 @@ function bytes(bytes, decimals, kib, maxunit) {
   return `${parseFloat((bytes / Math.pow(k, i)).toFixed(dm))} ${sizes[i]}`;
 }
 
+const darkModeMediaQuery = window.matchMedia('(prefers-color-scheme: dark)');
```

Comment:
Just checking this section aligns to our use of `media` for automatic dark/light mode?

Reply:
I guess this is not needed as we defined media auto light dark here: https://github.com/wg-easy/wg-easy/blob/master/src/tailwind.config.js

Reply:
I didn't quite understand how it works currently :)
The added query is needed to define colors of the charts for light / dark mode.

I can't find where `toggleTheme` method is used, and nothing happens when setting `theme` value in local storage to either `light` or `dark`.

There probably should be a toggle in the UI. The media query could be used to get current browser theme and then switch it with the toggle if needed.